src/@carbon-info/views/Data/Data.tsx

```diff
@@ -1,217 +1,243 @@
-            Cosmos SDK
-          </Typography>
-        </div>
-        <Typography variant="h2" color="textPrimary" className={clsx(classes.headerText, { open: inView })}>
-          Carbon allows anyone to bootstrap
-          <br />
-          open financial markets for
-          <br />
-          any asset type on any blockchain.
-        </Typography>
-      </FadeAndSlide>
-      <FadeAndSlide visible={inView} className={classes.dataWrapper}>
-        <AnimateKeyframes
-          play={inView}
-          iterationCount={1}
-          keyframes={["transform: scale(0)", "transform: scale(1)"]}
-          duration={0.75}
-        >
-          <Grid container item xs={8} sm={12} spacing={0} justifyContent="center" className={clsx(classes.dataTable, { open: inView })}>
-            {tableInfo.map((item: DataInfo) => (
-              <Grid item xs={12} sm={4} xl={2} key={item.description}>
-                <Paper className={classes.dataBox} elevation={0}>
-                  {item.toCountUp ? <RollingNum item={item} /> : <Typography variant="h3" color="textPrimary" align="center">{item.value}</Typography>}
-                  <Typography variant="body2" color="textSecondary" align="center">{item.description.replaceAll("_", " ")}</Typography>
-                </Paper>
-                {widthXs && <Divider className={classes.mobileDivider} />}
-              </Grid>
-            ))}
-          </Grid>
-        </AnimateKeyframes>
-      </FadeAndSlide>
-    </div >
-  );
+	return (
+		<div ref={ref} id="data" className={classes.container}>
+			<FadeAndSlide visible={inView} className={classes.dataWrapper}>
+				<AnimateKeyframes
+					play={inView}
+					iterationCount={1}
+					keyframes={["transform: scale(0)", "transform: scale(1)"]}
+					duration={0.75}
+				>
+					<Grid
+						container
+						item
+						xs={8}
+						sm={12}
+						spacing={0}
+						justifyContent="center"
+						className={clsx(classes.dataTable, { open: inView })}
+					>
+						{tableInfo.map((item: DataInfo) => (
+							<Grid item xs={12} sm={2} key={item.description}>
+								<Paper className={classes.dataBox} elevation={0}>
+									{item.toCountUp ? (
+										<RollingNum item={item} />
+									) : (
+										<Typography variant="h4" color="textPrimary" align="center">
+											{item.description === "Block_Time" ? "< 2s" : item.value}
+										</Typography>
+									)}
+									{/* Description */}
+									<Typography
+										variant="body2"
+										color="textPrimary"
+										align="center"
+									>
+										{item.description.replaceAll("_", " ")}
+									</Typography>
+								</Paper>
+								{widthXs && <Divider className={classes.mobileDivider} />}
+							</Grid>
+						))}
+					</Grid>
+					<Box className={classes.bannerContainer}>
+						<Banner
+							icon={<DemexLogo />}
+							text="Ready to test drive Carbon?"
+							highlightedText="Trade, Borrow and Earn on Demex"
+							link="https://app.dem.exchange/"
```

Comment:
Instead of hardcoding the link, would be good to use the hardcodes on the `Path` variable.

Think what you're looking for is `Path.Socials.Demex`.